classes/Product.php

```diff
@@ -2226,7 +2238,19 @@ public function updateAttribute(
                 'default_on' => null !== $default,
                 'minimal_quantity' => null !== $minimal_quantity,
                 'available_date' => null !== $available_date,
-            ]);
+            ];
+            if (is_array($available_now)) {
```

Comment:
I think because these are new fields, you could strictly type them as array in the method and you wouldn't need these "is_array" checks.